chat.koishi.koishi-desktop.yml

```diff
@@ -0,0 +1,105 @@
+id: chat.koishi.koishi-desktop
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+  - org.freedesktop.Sdk.Extension.node22
+command: koi
+rename-appdata-file: chat.koishi.desktop.appdata.xml
+rename-desktop-file: chat.koishi.desktop.desktop
+rename-icon: koishi-app
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --share=network
+  - --filesystem=home
+  - --talk-name=org.freedesktop.Notifications
```

Comment:
this doesn't go through the portal?

Reply:
This is related to the tray icon functionality. Since the app runs in the background and is configured via the web console, the tray icon provides quick access and visibility for managing it.

Reply:
I'm talking about notifications...